extern/boost/boost.cmake

```diff
@@ -44,7 +44,9 @@ ExternalProject_Add(
     LOG_CONFIGURE TRUE
     LOG_BUILD TRUE
     LOG_INSTALL TRUE
-    DOWNLOAD_NO_PROGRESS 1
+    DOWNLOAD_NO_PROGRESS 0
```

Comment:
Why is progress display turned off for some operations and on for others?

Reply:
depends on the tar file size, show the progress in large file downloading only